meeting-notes/2019/Q2/2019-06-03--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,213 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 June 3, 2019 
+
+- **Lead:** @jacobheun
+- **Notetaker:** @vasco-santos
+- **Attendees:**
+  - @vasco-santos
+  - @achingbrain
+  - @vmx
+  - @jacobheun
+  - @jimpick
+  - @hugomrdias
+  - @dirkmc
+  - @lidel
+  - @daviddias
+- **Recording:** https://youtu.be/ApZuiMgI9Ls
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@jacobheun
+ - Done:
+   - IPFS Camp preparation for libp2p workshop
+   - Started AutoNAT spec
+   - Helped with ipfs interop issue for latest go RC, https://github.com/ipfs/interop/pull/70
+     - Resulted in creating [libp2p-switch#339](https://github.com/libp2p/js-libp2p-switch/issues/339). We should handle dial backoff (blacklisting) by multiaddr instead of by peer as addresses can change over time.
+ - Blocked:
+ - Next: 
+   - More prep for IPFS Camp
+   - Finish Draft of AutoNAT spec and open PR
+   - Possibly continue with AutoNAT work in JS (time permitting)
+
+@vasco-santos
+- Done:
+  - enable browser tests on all pubsub modules
+    - [ChainSafe/gossipsub-js#28](https://github.com/ChainSafe/gossipsub-js/pull/28)
+    - [libp2p/js-libp2p-pubsub#19](https://github.com/libp2p/js-libp2p-pubsub/pull/19)
+    - [libp2p/js-libp2p-floodsub#80](https://github.com/libp2p/js-libp2p-floodsub/pull/80)
+  - update floodsub benchmarks
+    - [libp2p/js-libp2p-floodsub#82](https://github.com/libp2p/js-libp2p-floodsub/pull/82)
+  - update `libp2p` interop with new `go-libp2p`
+    - [libp2p/npm-go-libp2p-dep#3](https://github.com/libp2p/npm-go-libp2p-dep/pull/3)
+    - [libp2p/npm-go-libp2p-dep#6](https://github.com/libp2p/npm-go-libp2p-dep/pull/6)
+    - [libp2p/interop#18](https://github.com/libp2p/interop/pull/18)
+- Blocked:
+  - IPNS test [ipfs/interop#71](https://github.com/ipfs/interop/issues/71)
+- Next:
+  - add interop tests for hyprid pubsub network (needs to be implemented in gossipsub)
+    - [libp2p/interop#17](https://github.com/libp2p/interop/pull/17)
+  - integrate `interface-connection` proposal
+    - [libp2p/js-libp2p-tcp#102](https://github.com/libp2p/js-libp2p-tcp/pull/102)
+    - [libp2p/js-libp2p-websockets#82](https://github.com/libp2p/js-libp2p-websockets/pull/82)
+ - more async iterators work
```

Comment:
@vasco-santos can you help us out with shipping delegated content routing and shipping the DHT? It would be rad if you could handle re-providing?

Reply:
I will try to get `reprovide` during this week!